### PR TITLE
Userland/TaskbarWindow: Show context menu on right click

### PR DESCRIPTION
**Pull request by nyyakko**
**had to open the pr again because it seems like i messed up my previous fork? 😅😅

Just discovered and tried out the system today, and then I stumbled upon the fact that it didnt showed the classical context menu when i right clicked the taskbar. I found it quite unexpected, so i decided to implement one myself.

And thus this pr, which aims to add just that, came to be. It adds a simple context menu when the user clicks with the right button on the taskbar.

image

Right now its pretty simple and straight forward, but the idea can be easily expanded upon. Hope you guys like it :^)

### CODE REVIEW

File: Userland/Services/Taskbar/TaskbarWindow.cpp [hunk 1]
```diff
@@ -54,8 +65,44 @@ class TaskbarWidget final : public GUI::Widget {
...
+
+    auto show_desktop_action = GUI::Action::create("Show &Desktop", TRY(Gfx::Bitmap::load_from_file("/res/icons/16x16/desktop.png"sv)), [](auto&) {
+        GUI::ConnectionToWindowManagerServer::the().async_toggle_show_desktop();
+    });
+
+    auto open_settings_action = GUI::Action::create("&Settings", TRY(Gfx::Bitmap::load_from_file("/res/icons/16x16/settings.png"sv)), [this](auto&) {
+        GUI::Process::spawn_or_show_error(window(), "/bin/Settings"sv);
+    });
+
+    auto open_system_monitor_action = GUI::Action::create("&System Monitor", TRY(Gfx::Bitmap::load_from_file("/res/icons/16x16/app-system-monitor.png"sv)), [this](auto&) {
```
Comment (gmta) [inline Userland/Services/Taskbar/TaskbarWindow.cpp:93]: I think this leaves Settings and System Monitor with the same shortcut key 's'?

**Review by gmta — COMMENTED** [review]
Hello again :^) if you accidentally push your fork's branch without any commits, GitHub will automatically close the PR, so that's probably why your previous PR got nuked...

File: Userland/Services/Taskbar/TaskbarWindow.cpp (hunk 1 above)
Comment (nyyakko) [reply Userland/Services/Taskbar/TaskbarWindow.cpp]: ohhh! so thats what the ``&`` is for! 🤣fixed it!

**Review by kleinesfilmroellchen — APPROVED** [review]
The two commits can still be squashed, otherwise lgtm

**Review by kleinesfilmroellchen — APPROVED** [review]
Take another look at what the linting CI complains about

**Comment by bplaat** [comment]
Change the label because I want to see this get merged 😃

**Review by ADKaster — APPROVED** [review]

**Pull request merged**